watch-library/simulator/main.c

```diff
@@ -89,6 +89,7 @@ void main_loop_sleep(uint32_t ms) {
     main_loop_set_sleeping(true);
     emscripten_sleep(ms);
     main_loop_set_sleeping(false);
+    animation_frame_id = ANIMATION_FRAME_ID_INVALID;
```

Comment:
This is a seemingly unrelated change -- what is the point of it?

Reply:
You're right, it's probably the simulator workaround. Now that you mentioned it, I re-read the pull request text and it makes sense.

I don't think this is causing any problems on real hardware. It's only compiled in simulator builds.

Reply:
Oh, you're right - I just saw that this change is inside the simulator folder. All good, then!